### DISCUSSION TITLE
Difference in results for same model with OpenRadioss vs. Radioss 2021.1 due to contact interface T7 and T25 (probably?)

### DISCUSSION THREAD

**Discussion by rbianco-phi** [OpenRadioss/OpenRadioss#1919] [Q&A]
Hi everyone, 

I did some comparaison of OpenRadioss vs. Radioss 2021.1 and for highly complex models like mechanical ball joint model (sphere in a cup) with TYPE7 and TYPE25 contact interfaces, I see very different results (more than 5% in displacement). 

However, simulations are identical for "simple" cases such as tensile ou structural deformation, even for non-linear materials.

Has anyone noticed same results? 
Are contacts properties or parameters are taken into account differently for OpenRadioss?

I can't share my model, I have an NDA on this project.
Thanks you in advance.

Rohan.

**Comment by PaulAltair** [comment]
OpenRadioss does not do anything in contact differently to Commercial Radioss (it is the same source code), except, it is always newer! So it may have differences due to bugfixes etc. If you compare OpenRadioss with the latest commercial release (2022.3) then maybe they would be closer?, It is difficult to comment further or isolate a difference without knowing specifically the model or features. If you can reproduce on a non sensitive model then share by all means and we can try to isolate the differences.

If it is contact that is potentially causing the difference, for interface TYPE7, I would expect results to be similar between OpenRadioss and the 2021.1 commercial release, since TYPE 7 is a well established interface that hasn't had many updates recently, however if you have interface TYPE25, that was still quite new at the time of 2021.1 and has seen a number of bug fixes/developments over the last couple of years, so you may see more differences.

**Reply by elequiniou** [reply]
In such case, doing some sensitivity analysis with a given version/executable may help to understand if differences are coming from the effect of a software change or from the modeling and/or the physics involved.
Thanks,
Eric.
